Neos.Fusion/Classes/FusionObjects/AbstractArrayFusionObject.php

```diff
@@ -217,6 +231,8 @@ protected function isUntyped(string|int $key): bool
         if (!is_array($property)) {
             return false;
         }
-        return !isset($property['__objectType']) && !isset($property['__eelExpression']) && !isset($property['__value']);
+        return !array_key_exists('__objectType', $property)
+            && !array_key_exists('__eelExpression', $property)
+            && !array_key_exists('__value', $property);
```

Comment:
@mhsdesign because without this change the last test fails:
```patch
-'{"nullAttribute":null}'
+'{"nullAttribute":[]}'
```

Reply:
okay thanks ;) just wanted to make sure because changing sometimes to `array_key_exists` might have unintended side effects ;)

Reply:
Yes, good call but In this case it's an _intented_ side effect because for
```
Neos.Fusion:DataStructure {
   @apply.attributes = ${{nullAttribute: null}}
}
```
the property internally has the value
```json
{
  "__eelExpression": null,
  "__objectType": null,
  "__value": null,
}
```
so it _should_ be considered untyped, but it was _not_